src/payment/strategies/braintree/braintree.ts

```diff
@@ -88,6 +92,7 @@ export interface BraintreeThreeDSecureOptions {
     showLoader?: boolean;
     addFrame(error: Error | undefined, iframe: HTMLIFrameElement): void;
     removeFrame(): void;
+    onLookupComplete(data: any, next: any): void;
```

Comment:
Prefer to have definitive types rather than `any`.

Reply:
Ok, i've added new interface according to https://braintree.github.io/braintree-web/current/ThreeDSecure.html#~verificationData